File: ucluk/ucluk.c
#include <stdio.h>

int main()
{
    int arr1[3];
    int arr2[3];
    int num;
    int i=0, k=0;
    
    for(; i < 3; i++)
    {
        scanf("%d",&num);
        arr1[i] = num;
    }i = 0;

    for(; k < 3; k++)
    {
        scanf("%d",&num);
        arr2[k] = num;
    }
    
    int alice=0, bob=0;

    for(; i < 3; i++)
    {
        if(arr1[i] > arr2[i])
            alice++;
        else if(arr1[i] < arr2[i])
            bob++;
    }

    printf("%d  %d", alice, bob);
}

File: ucluk/ucluk.py
alice = list(map(int, input().split(" ")))
bob = list(map(int, input().split(" ")))

al = 0
bo = 0

for i in range(0,3):
    if alice[i] > bob [i]:
        al += 1
    elif bob[i] > alice[i]:
        bo += 1

print("{} {}".format(al,bo))
